kombu/utils/amq_manager.py

```diff
@@ -1,22 +1,34 @@
 """AMQP Management API utilities."""
 
-
 from __future__ import annotations
 
+from typing import Any
+
+from kombu import Connection
+
+DEFAULT_RABBITMQ_PORT = 15672
+
 
-def get_manager(client, hostname=None, port=None, userid=None,
-                password=None):
+def get_manager(
+    client: Connection,
+    hostname: str | None = None,
+    port: int | None = None,
+    userid: str | None = None,
+    password: str | None = None
```

Comment:
Based on PEP 604 Unions in annotations are supported from python version 3.10. Kombu currently is supporting also older versions of python se we need to use here `Optional`

Reply:
As we are using `from __future__ import annotations`, we don't. As annotations are not evaluated at runtime.

Reply:
this is OK. we can use python 3.10 stuffs in typing